VERSION

```diff
@@ -1 +1 @@
-1.231.0
\ No newline at end of file
+1.231.0//x-release-please-version
```

Comment:
This is not a JS file and those comment characters could be read by processing

Reply:
VERSION is being removed in PR 502.